app/main.go

```diff
@@ -27,22 +28,25 @@ func listSAs()([]LoadedIKE, error){
 	if err != nil {
 		return retVar, err
 	}
-	for _,m := range msgs.Messages() {
-		if e := m.Err(); e != nil{
+	for _, m := range msgs { // <- Directly iterate over msgs
```

Comment:
Do you know, why msgs.Messages() does not bring the expected result?
I didn't see any changes in the api regarding this, but i may be blind.

Reply:
I do not, but this appears to get the job done. Odd change I agree!